lib/parse-args.js

```diff
@@ -191,18 +189,13 @@ function buildYargs (withCommands = false) {
 function hideInstrumenterArgs (yargv) {
   let argv = process.argv.slice(1)
   argv = argv.slice(argv.indexOf(yargv._[0]))
-  if (argv[0][0] === '-') {
-    argv.unshift(process.execPath)
-  }
   return argv
 }
```

Comment:
This condition was never reached per the former code coverage report.

What use case does this line address?  I didn't find any documentation on passing a `-` for options via the command line.  

From what I can tell, if the first character of the second argument is a `-` then add the path of where C8 is located to the end of the argument array.  What purpose does this serve?  Do I need to add a Test Case instead of removing the code?